### DISCUSSION TITLE
Details

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#218] [Component suggestions]
<img width="480" alt="visuele weergave van de details component" src="https://github.com/user-attachments/assets/8a16ef38-80ba-42ee-9337-e8f2d76ad614" />

We willen graag meer leren over het **Details** component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

---

## Naam
Details

## Doel
Biedt de mogelijkheid om meer gedetailleerde informatie te tonen en weer te verbergen.

## Gerelateerde componenten
[Accordion](https://github.com/orgs/nl-design-system/discussions/217)

## Onderzoek

[`<details>`: The Details disclosure element - MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details)
Documentatie van Mozilla Developers Network over het `<details>` HTML element.

Zie [comment van Mike Alders bij Accordion component](https://github.com/orgs/nl-design-system/discussions/217#discussioncomment-8418984). In een [prototype voor Erfpacht](https://www.figma.com/file/jvKCJhmZGKv0fVJxSbxZ8v/Erfpacht---MijnDenHaag-%2F-Amsterdam?type=design&node-id=7501-12329&mode=design&t=9EJKcEsKzZYgRzLX-4) werd enkel Toon/Verberg niet altijd begrepen. Onderzoek zal geplaatst worden op gebruikersonderzoeken.nl maar vooralsnog te vinden via [Slack bericht van Nico Druif](https://codefornl.slack.com/archives/C7KDQRXMZ/p1707223988909559).

[Details - GOV.UK](https://design-system.service.gov.uk/components/details/)
Onder 'Research on this component' worden bevindingen genoteerd. Er is ook een [link naar de Details Github discussion](https://github.com/alphagov/govuk-design-system-backlog/issues/44) waar meer informatie onderzoek en aanpassingen worden gedeeld.

In de Discussion wordt vermeld dat uit onderzoek blijkt dat men de Details compont als Link ziet. Let wel: de Details component bij GOV heeft voorafgaand aan de tekst een icoon in de vorm van een gevulde driehoek die met de punt naar de tekst wijst. Daarnaast wordt er een [screenshot toegevoegd](https://github.com/alphagov/govuk-design-system-backlog/issues/44#issuecomment-629122091) waarbij er meerdere Detail componenten onder elkaar staan.

Ook wordt er vermeld dat de component niet door elke screenreader goed wordt voorgelezen. Daar op aanhakend wordt de volgende url gedeeld: https://u-elements.github.io/u-elements/elements/u-details

**Comment by jeffreylauwers** [comment]
## Amsterdam

[Figma](https://www.figma.com/file/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?type=design&node-id=5562-39900&mode=design&t=Buc9qlV18QiRDk8A-4)

---

### Screenshots / Varianten

**Figma bibliotheek**
![Scherm­afbeelding 2024-02-15 om 13 37 54](https://github.com/nl-design-system/backlog/assets/545207/a0ef716d-c8ea-4333-980b-1c59a654e345)

**Prototype Erfpacht**
![Scherm­afbeelding 2024-02-15 om 13 40 06](https://github.com/nl-design-system/backlog/assets/545207/ab39dc2e-b510-4665-b4e3-8647b8b576c9)

---

### Notities
- Zie 'Onderzoek' in [initieel discussion bericht](https://github.com/orgs/nl-design-system/discussions/218#discussion-6221398) voor meer informatie

**Comment by jeffreylauwers** [comment]
## DUO

[Documentatie website](https://uno.dfront.rijkscloud.nl/#/components/collapse)

---

### Screenshots / Varianten

**Collapse zonder achtergrond**
In een applicatie met minder ruimte of wanneer er minder focus vereist is, kan de kleinere collapse zonder achtergrond gebruikt worden.
![Scherm­afbeelding 2024-02-15 om 14 38 41](https://github.com/nl-design-system/backlog/assets/545207/17baf9dd-9016-4477-ae2c-b593fa7fbd9e)

---

### Notities
Onder de naam: Collapse (als variant zonder achtergrond). Collapse als geheel opgenomen in [Accordion discussion](https://github.com/orgs/nl-design-system/discussions/217#discussioncomment-8440754).

**Comment by jeffreylauwers** [comment]
## RVO

- [Storybook](https://nl-design-system.github.io/rvo/?path=/docs/componenten-expandable-content--documentatie)
- [GitHub](https://github.com/nl-design-system/rvo/tree/master/components/expandable-content)
- [Figma](https://www.figma.com/file/Sj6myBL1Fvot5M1qGxzvEo/ROOS-(RVO-Design-System)?type=design&node-id=4224-18830&mode=design&t=q6ESvHuatvkYvlV1-4)

---

### Screenshots / Varianten

![Scherm­afbeelding 2024-02-15 om 15 00 18](https://github.com/nl-design-system/backlog/assets/545207/41495b5a-b287-4154-b6a0-fc19041f920f)

---

### Notities
Onder de naam: Expandable content

**Comment by thijsren** [comment]
Hey, als ik het goed begrijp is dit component bedacht om meer informatie kwijt te kunnen over bepaalde onderwerpen. 
Bij Den Haag hebben we onlangs een soort gelijk component ontworpen met een iets andere aanpak. In plaats van de extra informatie inline te tonen tonen we het in een sheet of modal.

De voornaamste reden hiervoor is om focus te kunnen creëren, en kunnen we wanneer mogelijk is ook iets langere uitleg laten zien. Het component is initieel bedacht om in formulieren te gebruiken. Ook willen we 'uitklapper' in 'uitklapper' vermijden.

Het idee is ontstaan uit app-design, zoals Wise en Bux waarin ze een letterlijke uitleg functie hebben voor bepaalde termen (glossary).

Het idee is om het label zo goed mogelijk aan te laten sluiten op de het onderwerp.

<img width="698" height="687" alt="Screenshot 2026-01-22 at 16 41 36" src="https://github.com/user-attachments/assets/d2cdbc35-fb9d-4717-a33d-89922035e080" />

**Comment by jeffreylauwers** [comment]
## Expertteam: Cookie Consent patroon

Tijdens het uitwerken van een schets voor het Cookie Consent patroon had ik ook behoefte aan een Details-achtig component. Het betreft een call-to-action om aanvullende informatie over een group aan cookies te kunnen bekijken. Deze informatie is echt bedoelt voor mensen die de 'diepte' in willen wat betreft de specifieke cookies uit de groep. En waarschijnlijk voor de meeste gebruikers niet nodig om te selecteren welke groep van cookies zij wel of niet willen laten accepteren. Het gebruiken van 1 Accordion item zou hierbij onnodig veel aandacht trekken.

<img width="2379" height="1098" alt="Screenshot van pagina waarbij men Cookies kan instellen. Er zijn drie Cookies secties zichtbaar ieder met een checkbox. Onder de checkbox staan korte aanvullende informatie gevolgd door een Details component dat bestaat uit een icoon met de tekst 'Bekijk details'. Er is ook een annotatie sticker zichtbaar met de tekst 'Visually hidden text: Bekijk details voor negen analytische Cookies." src="https://github.com/user-attachments/assets/2a023a17-f7ec-4d1f-a560-3001c7941bdc" />

**Comment by jeffreylauwers** [comment]
## Expertteam: Alt-tekst Wizard

Tijdens het uitwerken van een schets voor de Alt-tekst Wizard had ik behoefte aan een Details-achtig component. Het betreft een call-to-action om aanvullende informatie over soorten-afbeeldingen te kunnen lezen. Deze informatie komt vooraf aan een formulier en waarschijnlijk voor de meeste gebruikers niet nodig om een keuze te maken. Het gebruiken van 1 Accordion-item zou hierbij onnodig veel aandacht trekken.

<img width="1586" height="1032" alt="Scherm­afbeelding 2026-02-23 om 14 35 43" src="https://github.com/user-attachments/assets/08ddc494-0ee3-4f79-a65a-34f4ac0317b5" />
